tests/system/src/test/java/test/robot/javafx/scene/MouseLocationOnScreenTest.java

```diff
@@ -120,6 +135,7 @@ private static void edge(Robot robot, int x1, int y1, int x2, int y2) {
         for (int x = x1; x <= x2; x++) {
             for (int y = y1; y <= y2; y++) {
                 robot.mouseMove(x, y);
+                delay(delayTime);
```

Comment:
Minor: you can use `Util.sleep` here instead of creating your own method.

Reply:
Fixed